### PR TITLE
start lightcontroller on raw3-3 bringup

### PR DESCRIPTION
**Pull request by benmaidel**

### CODE REVIEW

File: cob_bringup/robots/raw3-3.xml [hunk 1]
```diff
@@ -66,10 +66,10 @@
 			<arg name="component_name" value="torso" />
 			<arg name="robot" value="$(arg robot)" />
 		</include>
-                <include file="$(find cob_bringup)/controllers/cob_trajectory_controller.launch" >
-                        <arg name="component_name" value="torso" />
-                        <arg name="robot" value="$(arg robot)" />
-                </include>
+        <include file="$(find cob_bringup)/controllers/cob_trajectory_controller.launch" >
```
Comment (floweisshardt) [inline cob_bringup/robots/raw3-3.xml]: tabs vs spaces?

**Comment by benmaidel** [comment]
tabified the file

**Comment by floweisshardt** [comment]
visual review: OK

**Comment by abubeck** [comment]
LGTM

**Pull request merged**